### PR TITLE
Payara-1888 cdi probe documentation

### PR DESCRIPTION
**Pull request by OndrejM**
Gitbook is very slow today. Once the docs are generated, they will be available here: https://docs.payara.fish/v/PAYARA-1888_weld_probe/documentation/extended-documentation/development-tools/

### CODE REVIEW

**Comment by mikecroft** [comment]
Why is it in the the main repo not your own? Gitbook hasn't picked up any update on the branch, anyway.

**Comment by ggam** [comment]
Since this is a Weld specific feature it would be better said "Weld Probe" over "CDI Probe".

**Comment by OndrejM** [comment]
@mikecroft I use the main repo so that in case I'm not available, anyone can edit the PR. If done in my fork, others could either merge and then update or wait for me to update the PR with commits in my fork.

I also tried to push to my fork, but gitbook is very slow today, none of the commits except the very first one today has finished in hours.

**Comment by OndrejM** [comment]
@ggam Obviously, the tool comes from Weld and it hasn't been modified, only integrated. And we don't try to hide it, it's even stated in the documentation. 

However, Payara Server users may not know that Weld is used as the CDI engine and we decided to call the tool **CDI Probe** and the development mode **CDI Development Mode** to make it transparent and easy to understand its purpose.

**Comment by ggam** [comment]
Thanks for the clarification.

Just to avoid confusion, I didn't mean you tried to hide it or promote it
as a Payara featuere in any way. I just thought you just didn't noticed it.

File: documentation/extended-documentation/development-tools/cdi-dev-mode/cdi-probe/README.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+With CDI Development Mode enabled, Payara Server administration console provides access to the CDI Probe tool. This tool allows inspecting and monitoring CDI beans of deployed applications at runtime.
+
+CDI Probe provides the following interfaces:
+
+ - HTML-based application accessible via browser
+ - REST API
+ - JMX beans
+
+The CDI Probe tool is powered by the Weld CDI engine. More information can be found in the Weld project documentation about http://docs.jboss.org/weld/reference/2.3.5.Final/en-US/html/devmode.html[Development Mode].
```
Comment (OndrejM) [inline documentation/extended-documentation/development-tools/cdi-dev-mode/cdi-probe/README.adoc]: @ggam Weld is most prominently noticed here. In the rest of the documentation, we try to refer to CDI Probe if possible. I hope that's enough to give credit to Weld ;-)

**Comment by ggam** [comment]
Fine by me ;)

**Comment by mikecroft** [comment]
All looks good @OndrejM

**Comment by OndrejM** [comment]
@ggam Thank you for your support and feedback!

**Pull request merged**